### ISSUE TITLE
fix: `fatal: a failure has occurred: unexpected schema type render:services:WebServiceDetailsCreateRuntime `

### ISSUE THREAD

**Issue by VladimirCreator** [cloudy-sky-software/pulumi-render#367]
The error occurred after I executed the following command:

```bash
pulumi import render:services:WebService thewebsrcive srv-xxxxxxx
```

It also printed the following:

```
Your resource has been imported into Pulumi state, but there was an error generating the import code.
```

**Comment by VladimirCreator** [comment]
@praneetloke I now cannot update my infrastructure because it did not generate the code and now Pulumi thinks I deleted the resource and it tries to remove it 🫠

**Comment by praneetloke** [comment]
@VladimirCreator I suggest you temporarily remove the imported web service from your state with `pulumi state delete <urn> -s <stack name>`. You can get the URN with `pulumi stack --show-urns -s <stack name>`.

**Comment by praneetloke** [comment]
@VladimirCreator I had a suspicion that it was because the Render API for "Retrieve Service" does not return the `runtime` property and yet it is a required input for "Create Service", so Pulumi tries to use an empty string which ends up failing because `runtime` is an enum and it can't use an empty string for it. I proved this to be the case by hacking a local version of the provider where `runtime` is not an enum and Pulumi was able to generate the import code successfully.

Note the value of the `runtime` property in the screenshot.

![image](https://github.com/user-attachments/assets/761de7d7-8e5c-4f5e-8c43-8e7b56738468)

I have posted a [question](https://community.render.com/t/reporting-bugs-in-the-rest-api/25728) in their community forum but I am not sure if they'll fix their API. In the meantime, the only workaround that I can think of (aside from committing my hack where I change the type of `runtime` from an enum string to a regular string) is that you use this generated code from my import and adjust it to the values from your state after you run the import. I don't know how complex your web service configuration is though?

<details>
<summary>generated code</summary>

```ts
const myWebService = new render.services.WebService("myWebService", {
    autoDeploy: render.services.ServiceCreateAutoDeploy.Yes,
    branch: "main",
    name: "express-hello-world",
    ownerId: "usr-XXXXXXXXXXXX",
    repo: "https://github.com/render-examples/express-hello-world",
    rootDir: "",
    serviceDetails: {
        env: render.services.WebServiceDetailsCreateEnv.Node,
        envSpecificDetails: {
            buildCommand: "yarn --frozen-lockfile install",
            startCommand: "node app.js",
        },
        healthCheckPath: "",
        numInstances: 1,
        plan: render.services.WebServiceDetailsCreatePlan["free"],
        pullRequestPreviewsEnabled: render.services.WebServiceDetailsCreatePullRequestPreviewsEnabled.No,
        region: render.services.WebServiceDetailsCreateRegion.Oregon,
        runtime: "", <----this is what the CLI generated but you should fill this in with the value from the `env` property above.
    },
    type: "web_service",
}, {
    protect: true,
});
```
</details>

**Comment by praneetloke** [comment]
@VladimirCreator this should now be fixed with v0.3.12.
